recaptcha/src/create_key.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/bigtable/README.md
+ */
+
+namespace Google\Cloud\Samples\Recaptcha;
+
+// [START recaptcha_enterprise_create_site_key]
+use Google\Cloud\RecaptchaEnterprise\V1\RecaptchaEnterpriseServiceClient;
+use Google\Cloud\RecaptchaEnterprise\V1\Key;
+use Google\Cloud\RecaptchaEnterprise\V1\WebKeySettings;
+use Google\Cloud\RecaptchaEnterprise\V1\WebKeySettings\IntegrationType;
+
+/**
+ * Create a site key for reCAPTCHA
+ * @param string $projectId Your Google Cloud project ID
+ * @param string $keyName The name of the key you wish to create
+ */
+function create_key(
+    string $projectId,
+    string $keyName
+): void {
+    $client = new RecaptchaEnterpriseServiceClient();
+    $formattedProject = $client->projectName($projectId);
+
+    // create the settings for the key
+    // in order to create other keys we'll use AndroidKeySettings or IOSKeySettings
```

Comment:
nit, but use sentences
```suggestion
    // Create the settings for the key.
    // In order to create other keys we'll use AndroidKeySettings or IOSKeySettings.
```